compositeviews/compositeview01/researchproject.json

```diff
@@ -0,0 +1,93 @@
+{
+  "@id": "connectome-es-projection-researchproject-01",
+  "@type": "ElasticSearchProjection",
+  "mapping": {
+    "dynamic": false,
+    "properties": {
+      "@id": {
+        "copy_to": "_all_fields",
+        "type": "keyword"
+      },
+      "@type": {
+        "copy_to": "_all_fields",
+        "type": "keyword"
+      },
+      "name": {
+        "copy_to": "_all_fields",
+        "type": "text"
+      },
+      "description": {
+        "copy_to": "_all_fields",
+        "type": "text"
+      },
+      "description_en": {
+        "copy_to": "_all_fields",
+        "type": "text"
+      },
+      "description_de": {
+        "copy_to": "_all_fields",
+        "type": "text"
+      },
+      "description_fr": {
+        "copy_to": "_all_fields",
+        "type": "text"
+      },
+      "description_it": {
+        "copy_to": "_all_fields",
+        "type": "text"
+      },
+      "member": {
+        "properties": {
+          "name": {
+            "type": "text"
+          },
+          "givenName": {
+            "type": "text"
+          },
+          "familyName": {
+            "type": "text"
+          }
+        },
+        "type": "nested"
+      },
+      "_all_fields": {
+        "copy_to": "_all_fields",
+        "type": "text"
+      }
+    }
+  },
+  "query": "es_projection_query.rq",
```

Comment:
Same note as above regarding the "templating syntax" of this JSON

Reply:
see #29